queue_job_context/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Queue Job, prepare context before enqueue keys
```

Comment:
```suggestion
Set the list of context keys that are propagated to jobs ("tz", "lang", "allowed_company_ids", "force_company", "active_test"). This will become the default in 16.0.
```

Reply:
done